toonz/sources/toonz/mainwindow.cpp

```diff
@@ -2031,8 +2031,8 @@ void MainWindow::defineActions() {
   createToolOptionsAction("A_ToolOption_PencilMode", tr("Pencil Mode"), "");
   createToolOptionsAction("A_ToolOption_PreserveThickness",
                           tr("Preserve Thickness"), "");
-  createToolOptionsAction("A_ToolOption_PressureSensitivity",
-                          tr("Pressure sensitivity"), "");
+  createToolOptionsAction("A_ToolOption_PressureSensibility",
```

Comment:
@turtletooth 
Oh, sorry for making you confused! Now I understand. -TIVITY is correct.
Then I think this line 2034 should be kept unchanged. Please forget my previous comment.